.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10
```

Comment:
Do we support only node 10?

Reply:
We were running on v6 and then migrated over to v8 when webtask upgrade node.js. I believe that repo supervisor should be compatible with Node.js v6+ all the way to the latest versions.

Reply:
Shouldn't we run the tests for all the major node versions we support?

Reply:
We might think about it in the future since it's not set in stone which versions we want to support yet. 👍

Reply:
Sounds good.